website/docs/r/cloud_backup_snapshot_export_job.html.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: cloud_backup_snapshot_export_job"
+sidebar_current: "docs-mongodbatlas-resource-cloud_backup_snapshot_export_job"
+description: |-
+    Provides a Cloud Backup Snapshot Export Job resource.
+---
+
+# mongodbatlas_cloud_backup_snapshot_export_job
+`mongodbatlas_cloud_backup_snapshot_export_job` resource allows you to retrieve all the buckets for the specified project. 
+
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+## Example Usage
+
+```terraform
+
+resource "mongodbatlas_cloud_backup_snapshot_export_bucket" "test" {
+  project_id = "{PROJECT_ID}"
+  iam_role_id    = "{IAM_ROLE_ID}"
+  bucket_name    = "example_bucket"
+  cloud_provider = "AWS"
+}
+
+resource "mongodbatlas_cloud_backup_snapshot_export_job" "test" {
+  project_id   = "{PROJECT_ID}"
+  cluster_name = "{CLUSTER_NAME}"
+  snapshot_id = "{SNAPSHOT_ID}"
+  export_bucket_id = mongodbatlas_cloud_backup_snapshot_export_bucket.test.export_bucket_id
+  
+  custom_data {
+    key   = "exported by"
+    value = "myName"
+  }
+}
+
+```
+
+## Argument Reference
+
+* `project_id` - (Required) Unique 24-hexadecimal digit string that identifies the project which contains the Atlas cluster whose snapshot you want to export.
+* `cluster_name` - (Required) Name of the Atlas cluster whose snapshot you want to export.
+* `snapshot_id` - (Required) Unique identifier of the Cloud Backup snapshot to export.
+* `export_bucket_id` - (Required) Unique identifier of the AWS bucket to export the Cloud Backup snapshot to.
```

Comment:
```suggestion
* `export_bucket_id` - (Required) Unique identifier of the AWS bucket to export the Cloud Backup snapshot to.  If necessary, use the [Get All Snapshot Export Buckets](https://docs.atlas.mongodb.com/reference/api/cloud-backup/export/get-all-export-buckets/) API to retrieve the IDs of all available export buckets for a project or use the data source [mongodbatlas_cloud_backup_snapshot_export_buckets](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/backup_snapshot_export_buckets)
```

Without this info from the docs and how to get from the data source the user will not know where to obtain these.  We must direct them to the right place.

Reply:
done